src/pages/iou/IOUDetailsModal.js

```diff
@@ -173,10 +175,28 @@ class IOUDetailsModal extends Component {
             });
     }
 
+    /**
+     * Checks to see if we can use Expensify.
```

Comment:
> Checks to see if we can use Expensify.

Could you expand on this a bit? 'use Expensify' without the pay-with-expensify context is a bit vague IMO.